test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -81,8 +81,8 @@ public void addCommand_invalidTags_throwsException() {
      * invalid data throws an IllegalValueException
      */
     private void assertConstructingInvalidAddCmdThrowsException(String name, String phone,
-            boolean isPhonePrivate, String email, boolean isEmailPrivate, String address,
-            boolean isAddressPrivate, Set<String> tags) {
+                                                                boolean isPhonePrivate, String email, boolean isEmailPrivate, String address,
+                                                                boolean isAddressPrivate, Set<String> tags) {
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. extra spaces.